### ISSUE TITLE
"Weekly news" updates

### ISSUE THREAD

**Issue by lalaithan** [diegocard/awesome-html5#20]
HTML5 Weekly is now Frontend Focus and Mozilla Hacks Weekly doesn't seem to be active since 12/20/12.

**Comment by ravleenkaur8368** [comment]
yes try next

**Comment by lalaithan** [comment]
> yes try next

Next?

**Comment by bikash1317** [comment]
Is this project open I can write the most as I am a front end developer . If it is still open just let me know .

**Comment by lalaithan** [comment]
> Is this project open I can write the most as I am a front end developer . If it is still open just let me know .

This issue isn't the place to ask that.

**Comment by diegocard** [comment]
Yes, the project is still open! Happy to receive pull requests for the HTML5 Weekly change.
